docs/src/API.md

```diff
@@ -27,8 +27,7 @@ Korg.rectify
 Korg.constant_R_LSF
 ```
 
-## Line absorption 
-These functions can be used to directly compute line opacities. 
+## Line absorption These functions can be used to directly compute line opacities. 
```

Comment:
I think this would create a really long entry in the table of contents on the left of the page.

Reply:
I agree. I think it would be better to have `## Line absorption` and `These functions can be used to directly compute line opacities.` on 2 separate lines.

In the current PR, you already concatenated them into a single line. Was that unintentional?